src/types/meta_item.rs

```diff
@@ -54,8 +54,18 @@ pub struct MetaPreview {
     #[serde(default)]
     pub name: String,
     pub poster: Option<String>,
+    pub logo: Option<String>,
+    pub description: Option<String>,
+    pub release_info: Option<String>,
+    pub runtime: Option<String>,
+    pub released: Option<DateTime<Utc>>,
```

Comment:
I'm not sure `released` and `release_info` should be there together
have you checked whether this is consistent with the SDK documentation?

also, please comment in the code that some fields are obsolete and should be removed, like `genres` and `directors`

Reply:
released is the date it was released.
release info is string like `2007-2009` or `2007-`

Reply:
genres and directors marked as deprecated